src/main/java/team/gif/robot/commands/telescopingArm/MotorRun.java

```diff
@@ -0,0 +1,46 @@
+package team.gif.robot.commands.telescopingArm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import team.gif.robot.Constants;
+import team.gif.robot.Robot;
+
+public class MotorRun extends CommandBase {
+    public MotorRun() {
+        super();
+        addRequirements(Robot.telescopingArm);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {}
+
+    // Called every time the scheduler runs (~20ms) while the command is scheduled
+    @Override
+    public void execute() {
+//        if (Robot.telescopingArm.getVelocity() == Constants.TelescopingArm.MAX_VELOCITY) {
+//            Robot.telescopingArm.setTelescopingMotor(0);
+//        } else {
+//            Robot.telescopingArm.setTelescopingMotor(Constants.TelescopingArm.TEST);
+//        }
+
+        double speed = -Robot.oi.aux.getLeftY();
```

Comment:
this will conflict with the manual arm control. I think we'll need to create a combo button press for the manual telescoping arm. Something like pushing in the right stick and moving the left stick ...
double speed = 0
if( Robot.oi.aux.getRightStickButton() && Robot.oi.aux.getLeftY() )
    speed = -Robot.oi.aux.getLeftY();